mathics/core/symbols.py

```diff
@@ -621,12 +621,18 @@ def __cmp(self, other) -> int:
 
 
 class Atom(BaseExpression):
-    """
-    Atoms are the leaves and heads of an Expression or S-Expression.
+    """Atoms are the leaves (in the common tree sense, not the Mathics
```

Comment:
It is common to put the comment in the next line:
```py
"""
Atoms are the leaves (in the common tree sense, not the Mathics
...
"""
```

Reply:
Actually in GNU Emacs python mode Esc-W "normal" is to join `"""` with what is above and reflow the paragraph. 

Someday I'll figure out how to reconcile GNU Emacs Python normal with Tiago's normal and this will be solved. 

In the meantime I've changed this manually.